introduction.md

```diff
@@ -0,0 +1 @@
+I am Yako and I am new to the dojo. It is an honor to me you.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.